ntoskrnl/rtl/libsupp.c

```diff
@@ -37,6 +37,10 @@ RtlPcToFileHeader(
 {
     PLDR_DATA_TABLE_ENTRY LdrEntry;
     BOOLEAN InSystem;
+    KIRQL OldIrql;
+
+    /* Acquire the loaded module spinlock */
+    OldIrql = KeAcquireSpinLockRaiseToSynch(&PsLoadedModuleSpinLock);
```

Comment:
Why raise to synch ?

Reply:
I guess it's not needed, I simply copied it from MiProcessLoaderEntry. I'll change to dispatch.